python/essex-config/tests/integration/test_graphrag_config.py

```diff
@@ -2,13 +2,14 @@
 from pathlib import Path
 from unittest import mock
 
+from essex_config import load_config
 from essex_config.sources import EnvSource, FileSource
 
 from .graphrag_config import GraphRagConfig, LLMType, ReportingType, TextEmbeddingTarget
```

Comment:
I know it is not in the context of this review but... Personally I don't like so much GraphRAG stuff in a project that's not related. Conceptually it can be anything, I understand, but, seems like it is pulling too much unnecessary stuff for just a test. 
If anything, this should be in GraphRAG to test the GraphRAG config once we adopt essex-config, not the other way around.

Reply:
Create an issue #277 to discuss this out of this PR